scripts/git/pre-commit

```diff
@@ -80,12 +82,27 @@ def main():
             print_objection()
 
         num_errors += len(output)
-        for error in output:
-            print "%s:%s: %s" % (filename, str(error[0]), error[1])
+        if len(output) > 0:
+            lines = []
+            for error in output:
+                print "%s:%s: %s" % (filename, str(error[0]), error[1])
+                lines.append(error[0])
+            # Run the clang-format if there are errors in the identified ranges
+            if len(output) > 0:
+                command = "clang-format-8 -style=file -i " + filename
```

Comment:
How is clang-format-8 installed? Need to add to install_desktop_packages?

Reply:
yes, I added clang-format-8 to the packages_desktop.lst because this is a package that does not go in the robot.